tests/test_core.py

```diff
@@ -87,8 +91,14 @@ def test_core_count_missing_data(tmp_path, mock_db):
         ref_table = []
         for row in f.readlines():
             ref_table.append(eval(row))  # pylint: disable=eval-used
-    for row in ref_table:
-        assert row in table_rows
+    try:
+        for row in ref_table:
+            assert row in table_rows
+        for row in table_rows:
+            assert row in ref_table
+    except Exception as e:
+        conftest.debug_diff_tables(cols, table_rows, ref_table, pos=0)
+        raise e
```

Comment:
nit: You can simply say `raise` and it will re-raise the current exception. (which is nice, because then you don't even have to give the exception a name)

Reply:
huh! i had no idea.